### PR TITLE
feat: add content preference indexes

### PR DESCRIPTION
**Pull request by capJavert**
- adding indexes for feed queries that will be used, tried on queries and it looks like all are hit

### CODE REVIEW

File: src/migration/1731520135216-ContentPreferencesReadIndexes.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class ContentPreferencesReadIndexes1731520135216 implements MigrationInterface {
+    name = 'ContentPreferencesReadIndexes1731520135216'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+      await queryRunner.query(`CREATE INDEX IF NOT EXISTS "idx_content_preferences_referenceid_type_status" ON "content_preference" ("referenceId", "type", "status")`);
```
Comment (capJavert) [inline src/migration/1731520135216-ContentPreferencesReadIndexes.ts:7]: Already added all indexes in production with `CONCURRENTLY`

File: src/migration/1731520135216-ContentPreferencesReadIndexes.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+      await queryRunner.query(`CREATE INDEX IF NOT EXISTS "idx_content_preferences_userid_referenceid_type" ON "content_preference" ("userId", "referenceId", "type")`);
+
+      await queryRunner.query(`CREATE INDEX IF NOT EXISTS "idx_content_preferences_feedid_type_status" ON "content_preference" ("feedId", "type", "status")`);
+
+      await queryRunner.query(`CREATE INDEX IF NOT EXISTS "idx_content_preferences_feedid_type_userid_status" ON "content_preference" ("feedId", "type", "userId", "status")`);
+    }
+
+    public async down(queryRunner: QueryRunner): Promise<void> {
```
Comment (omBratteng) [inline src/migration/1731520135216-ContentPreferencesReadIndexes.ts:16]: Add IF EXISTS to thesee

**Review by omBratteng — CHANGES_REQUESTED** [review]

**Review by rebelchris — APPROVED** [review]
Nothing besides Ole-Martin comment

File: src/migration/1731520135216-ContentPreferencesReadIndexes.ts (hunk 2 above)
Comment (capJavert) [reply src/migration/1731520135216-ContentPreferencesReadIndexes.ts:16]: done!

**Pull request merged**